<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>245</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>541</width>
     <height>241</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>521</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Website Details</string>
     </property>
     <widget class="QLineEdit" name="auth_code">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>441</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Auth Code</string>
      </property>
     </widget>
     <widget class="QComboBox" name="driver_list">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>441</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>47</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Driver</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>251</width>
       <height>110</height>
      </rect>
     </property>
     <property name="title">
      <string>Forwarding</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Host</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>47</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Port</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="forwarding_host">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>160</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="forwarding_port">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>160</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="forwarding_enable">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>160</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="local_mode_box">
     <property name="geometry">
      <rect>
       <x>269</x>
       <y>100</y>
       <width>261</width>
       <height>110</height>
      </rect>
     </property>
     <property name="title">
      <string>Local Mode</string>
     </property>
     <widget class="QPushButton" name="local_mode">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>241</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>241</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Local mode disables the sending of lap and session data to the RLC website</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Telemetry</string>
    </attribute>
    <widget class="QCheckBox" name="telemetry_enable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>140</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Telemetry Window</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Center Value</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Bottom Right Value</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Bottom Left Value</string>
     </property>
    </widget>
    <widget class="QComboBox" name="center_value">
     <property name="geometry">
      <rect>
       <x>119</x>
       <y>50</y>
       <width>410</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Speed Delta</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fuel Remaining</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lap Delta</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="bottom_left_value">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>410</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Lap Delta</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fuel Remaining</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Speed Delta</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="bottom_right_value">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>410</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Fuel Remaining</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Speed Delta</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lap Delta</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="diff_value">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>410</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Last Lap</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Best Lap</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Difference based on</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>CodeMasters F1</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>521</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Enable Telemetry</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QPushButton" name="f1_2016">
      <property name="geometry">
       <rect>
        <x>265</x>
        <y>20</y>
        <width>245</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>F1 2016</string>
      </property>
     </widget>
     <widget class="QPushButton" name="f1_2015">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>245</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>F1 2015</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>59</y>
        <width>500</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Click the above buttons to enable or disable the telemetry system for the respective game. If the location of the configuration file is unknown, you will be prompted to find it. The filename to select is called 'hardware_settings_config.xml'.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>rFactor2</string>
    </attribute>
    <widget class="QPushButton" name="rf2_install">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>521</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Install</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>511</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>The above button will open a dialog box - please locate your rfactor2 installation directory. After this has been selected, a plugin will be installed that will allow this app to receive select telemetry data (speed, fuel, sector and lap times).</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Assetto Corsa</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Project Cars</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>iRacing</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
